server_admin/topics/clients/oidc/con-advanced-settings-ui.adoc

```diff
@@ -0,0 +1,126 @@
+[id="con-advanced-settings_{context}"]
+= Advanced settings
+[role="_abstract"]
+When you click the *Advanced* tab, additional fields are displayed.
+
+*Revocation* and *Clustering* are topics described in the link:{installguide_link}#_clustering[{installguide_name}]
```

Comment:
That installation guide is no longer part of the keycloak documentation that is built.  The new version is here.
https://www.keycloak.org/guides#server

However, I don't think it covers those topics.  So it is best to remove this statement.